apps/cli/src/commands/boot-challenger.ts

```diff
@@ -258,7 +311,9 @@ async function processMissedAssertions(commandInstance: Vorpal.CommandInstance)
             await submitAssertionToReferee(
                 cachedSecretKey,
                 missedAssertionNodeNum,
-                assertionNode,
+                missedAssertionNodeNum - 1,
+                assertionNode.confirmData,
+                assertionNode.createdAtBlock,
```

Comment:
There is more consideration to be done with `processMissedAssertions`, this could lead in us submitting the missed challenge immediately.